### PR TITLE
profile/account settings

### PR DESCRIPTION
**Pull request by jordanepickett**
added the ability to change name, email, and password in the newly named
Account Settings.

### CODE REVIEW

File: app/Http/Controllers/Web/Profile/ProfileSettingsController.php [hunk 1]
```diff
@@ -26,7 +28,58 @@ class ProfileSettingsController extends Controller
...
+                    Messages::error('Password is incorrect.');
+
+                    return $this->redirect(route('profile.index'));
+                }
+                $password = Hash::make($input['newPassword']);
+                user()->password = $password;
+            }
+
+            user()->save();
+
```
Comment (jaketoolson) [inline app/Http/Controllers/Web/Profile/ProfileSettingsController.php:81]: Let's fire an event after save called something like `UserSettingsUpdated` and pass it the `user()` as the only param.

File: app/Http/Controllers/Web/Profile/ProfileSettingsController.php [hunk 2]
```diff
@@ -26,7 +28,58 @@ class ProfileSettingsController extends Controller
...
+            'newPassword_confirmation' => Binput::get('newPassword_confirmation')
+        ];
+
+        // Set Validation Rules
+        $rules = [
+            'name' => 'required',
+            'email' => 'required|email|unique:users,email,' . user()->id,
+            'password' => 'required_with:newPassword,newPassword_confirmation',
+            'newPassword' => 'required_with:newPassword_confirmation,password|min:6|confirmed',
+            'newPassword_confirmation' => 'required_with:newPassword_confirmation'
```
Comment (jaketoolson) [inline app/Http/Controllers/Web/Profile/ProfileSettingsController.php]: I believe this should be `required_with:newPassword` right?

Comment (jordanepickett) [reply app/Http/Controllers/Web/Profile/ProfileSettingsController.php]: oops, you're right. lemme fix that

**Comment by jaketoolson** [comment]
@jordanepickett  Got another request for this: We need an area to set the user's timezone :) Perhaps using this to generate the list they can select from, and then the default should always be `America/Los_Angeles`. Thoughts?   We will need a DB migration that adds "timezone" field to the user table (string) with a default of `America/Los_Angeles`.

**Pull request merged**